config/locales/en.yml

```diff
@@ -1851,6 +1851,7 @@ en:
   # Label for the date/time when a flag was created
   flag_created: Flag Created
   flag_description: Flag this content to let the curators know something is wrong with it.
+  flag_comment_description: Add a comment to the flag to add supporting rationale for the change, such as links.
```

Comment:
My change! Does the community help translate these strings to every other language or should I do that with Google translate?

Reply:
Yeah, translation is done by other volunteers at https://crowdin.com/project/inaturalistweb. Strings from the main branch get synced to Crowdin, so this new string will be available for translation when we merge this.

Reply:
Let's change this to "Add an optional comment to this flag that further explains why you are adding it and provides links to supporting evidence."